docs_user/modules/openstack-keystone_adoption.adoc

```diff
@@ -12,6 +12,29 @@ must already be imported into the podified MariaDB.
 
 == Pre-checks
 
+== Copy over fernet keys
+
+Pull your [openstack configuration using os-diff](pull_openstack_configuration.md) from the old environment.
```

Comment:
AsciiDoc syntax - see line 6 above for local link syntax. But now that we pull the config directly, we can probably remove this line altogether?